### DISCUSSION TITLE
Featured Image

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#427] [General]
# Featured Image

This module shows the featured image above the post content if present.

https://hbstack.dev/en/modules/blog/featured-image/

**Comment by Productivix** [comment]
hi, thanks for this template .
I try to add a post with image - I copied the index.en.md of the other posts and put a different or the same image , I named it as described in this doc featured-01.jpg but I have a server error : 
'
ERROR Rebuild failed: render: failed to render pages: render of "page" failed: "/home/william/.cache/hugo_cache/modules/filecache/modules/pkg/mod/github.com/hbstack/blog@v0.15.6/layouts/blog/single.html:2:5": execute of template failed at <partial "hb/modules/blog/single" .>: error calling partial: "/home/william/.cache/hugo_cache/modules/filecache/modules/pkg/mod/github.com/hbstack/blog@v0.15.6/layouts/partials/hb/modules/blog/single.html:25:12": execute of template failed at <partial "hugopress/functions/render-hooks" (dict "Name" "hb-blog-post-content-begin" "Page" .)>: "/home/william/.cache/hugo_cache/modules/filecache/modules/pkg/mod/github.com/hugomods/hugopress@v0.2.1/layouts/partials/hugopress/functions/render-hooks.html:20:8": execute of template failed at <partial .partial $ctx>: "/home/william/.cache/hugo_cache/modules/filecache/modules/pkg/mod/github.com/hbstack/blog/modules/featured-image@v0.2.10/layouts/partials/hugopress/modules/hb-blog-post-featured-image/hooks/hb-blog-post-content-begin.html:1:4": execute of template failed at <partialCached "hb/modules/featured-image/index" .Page .Page>: error calling partialCached: execute of template failed at <partial "images/image" (dict "Page" $page "Alt" $page.Title "Filename" $img "ClassName" "hb-featured-img img-fluid p-1 bg-body-secondary border" "LazyLoading" false "Original" true)>: "/home/william/.cache/hugo_cache/modules/filecache/modules/pkg/mod/github.com/hugomods/images@v0.7.0/layouts/partials/images/image.html:72:17": execute of template failed at <.Resize>: error calling Resize: image "/home/william/logiciels/logiciels-windows/dev_RN-Node-GIT/site_productivix_hbstarter/content/blog/installapp-mobile/featured-01.jpg": resize : EOF
'
I do not understand the origin of the error.

**Comment by razonyang** [comment]
Could you please provide the image and example content for reproducing? It would be better if there is a repo for checking this issue, sometime it was related to the incorrect configuration.

It seems that the image cannot be resized in some reasons. Have you try to use another image?

**Comment by Productivix** [comment]
hi , thanks for the answer , I translated your original theme into French / English, I put it public on https://github.com/Productivix/site_productivix_hbstarter
the problem is on https://github.com/Productivix/site_productivix_hbstarter/tree/main/content/blog/installapp-mobile
If I change the image name : no blocking but image not displayed

**Comment by razonyang** [comment]
I was able to reproduce it with your image, seems Hugo cannot handle this one, but I have no clue on this.
It works when converting it to jpg via GIMP.

![inventaire2](https://github.com/hbstack/site/assets/17720932/a22c02d7-86c2-455e-bcc5-8adf670ff14c)

**Reply by Productivix** [reply]
ok - those images are made by linux standard screen capture -

**Reply by razonyang** [reply]
Would you mind I using this image to post an issue on the Hugo forum?
Btw, what software do you use for screenshot on Linux? I'll try to test it on my Arch Linux.

**Reply by Productivix** [reply]
Yes ok , use it  - 
the soft is https://docs.xfce.org/apps/screenshooter/start

**Reply by Productivix** [reply]
please tell me the link of the issue to follow also. thanks

**Comment by razonyang** [comment]
> please tell me the link of the issue to follow also. thanks

https://discourse.gohugo.io/t/cant-resize-special-images-unexpected-eof/45991

I'm closing this for now, let's keep tracking this issue on the forum.

**Comment by razonyang** [comment]
I took some screenshots via the XFCE4 screenshooter, but all of them works, it seems that this one is invalid/incorrect as mentioned on the forum.

**Comment by Productivix** [comment]
Hi, yes you are right : all my screenshot tools , including the one of XFCE4 produce excellent jpg images , so it could be in image coping / replications ! My image problems comes from an another tool ! Sorry . and thanks for the identification tool : 
Using ImageMagick’s identify tool:

$ identify -verbose inventaire.jpg
